fec_eregs/static/fec_eregs/js/data/terms.json

```diff
@@ -6,6 +6,10 @@
   {
     "term": "Administrative expense",
     "definition": "For party committees, rent, utilities, office equipment, office supplies, routine building maintenance and other operating costs not attributable to a specific candidate."
+  },
+    {
+    "term": "Adminstrative termination",
+    "definition": "Administrative termination is the action taken by the Commission to terminate the reporting obligations of political committees that appear to be inactive in accordance with 52 U.S.C.<a href=\" https://www.govinfo.gov/link/uscode/52/30103\">30103(d)(2)</a> and <a href=\"https://www.fec.gov/regulations/102-4/CURRENT#102-4\">11 CFR 102.4</a>. Administrative termination does not relieve a committee of any legal responsibility for the payment of any outstanding debt or obligations."
```

Comment:
Nice work! For the first link, can we close the space in `href=\" https://www`? I'm not sure if it'll break anything to leave it, but better safe than sorry.

Reply:
Woo hoo! Thanks, @kathycarothers !